lldb/include/lldb/API/SBFunction.h

```diff
@@ -44,6 +45,8 @@ class LLDB_API SBFunction {
 
   lldb::SBAddress GetEndAddress();
 
+  lldb::SBAddressRange GetRange();
+
```

Comment:
Allow a function to have discontiguous ranges:
```
lldb::SBAddressRangeList GetRanges();
```

Reply:
GetRange() is just a forwarded function from lldb_private::Function::GetRange()

Reply:
Or did you want me to return SBAddressRangeList which will always contain only one SBAddressRange unless SBFunction ever changes?